cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -930,14 +928,13 @@ function Configure-GcePdTools {
 Unblock-File $modulePath
 Import-Module -Name $modulePath'.replace('K8S_DIR', ${env:K8S_DIR})
 
-  if (Test-IsTestCluster $kube_env) {
-    if (ShouldWrite-File ${env:K8S_DIR}\diskutil.exe) {
-      # The source code of this executable file is https://github.com/kubernetes-sigs/sig-windows-tools/blob/master/cmd/diskutil/diskutil.c
-      MustDownload-File -OutFile ${env:K8S_DIR}\diskutil.exe `
-        -URLs "https://ddebroywin1.s3-us-west-2.amazonaws.com/diskutil.exe"
-    }
-    Copy-Item ${env:K8S_DIR}\diskutil.exe -Destination "C:\Windows\system32"
+  if (ShouldWrite-File ${env:K8S_DIR}\diskutil.exe) {
+    # The source code of this executable file is https://github.com/kubernetes-sigs/sig-windows-tools/blob/master/cmd/diskutil/diskutil.c
+    MustDownload-File -OutFile ${env:K8S_DIR}\diskutil.exe `
+      -URLs "https://ddebroywin1.s3-us-west-2.amazonaws.com/diskutil.exe"
```

Comment:
it's kind of ok to download from the above URL if the cluster is for test, it is not ok if it is not for test.

Reply:
we recently added an API in csi-proxy which will remove the need of this diskutil. I will update driver and remove this part after testing.